Dockerfile.android

```diff
@@ -37,12 +42,12 @@ RUN dpkg --add-architecture i386 && \
 
 # Jenkins will run this image as user 1001, let's ensure that user exists and has a home directory
 RUN adduser --uid 1001 --disabled-password --gecos "" jenkins
-# Install the SDK and NDK as jenkins
+# Install the SDK and NDK
 USER jenkins
 
 # Install the Android SDK and NDK
 ENV ANDROID_HOME /home/jenkins/android-sdk
-ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
+ENV PATH ${PATH}:${ANDROID_HOME}/emulator:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
```

Comment:
Now that we're touching and adding to this, you might want to refactor this to no use the deprecated `ANDROID_HOME` environment variable. See https://developer.android.com/studio/command-line/variables#android_sdk_root.

Reply:
This might be worth it in a follow up PR.

Reply:
I remember now that I already added an issue tracking this https://github.com/realm/realm-js/issues/3477#issue-767565143